Sources/XCTest/TestFiltering.swift

```diff
@@ -42,12 +37,12 @@ internal struct TestFiltering {
 
     static func filterTests(entries: [XCTestCaseEntry], filter: TestFilter) -> [XCTestCaseEntry] {
         return entries
-            .map({ testCase, tests in
-                return (testCase, tests.filter({ filter(testCase, $0.0) }))
-            })
-            .filter({ testCase, tests in
+            .map { testCase, tests in
+                return (testCase, tests.filter { filter(testCase, $0.0) } )
+            }
+            .filter { testCase, tests in
                 return !tests.isEmpty
-            })
+            }
```

Comment:
I'm pretty ambivalent about this one. I've been pretty happy following the [Rule of Kevin](http://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/) in my Swift code, but I don't have a strong opinion.